.eslintrc.json

```diff
@@ -111,6 +111,7 @@
     "implicit-arrow-linebreak": "off",
 
     "eslint-comments/no-unused-disable": "warn",
+    "eslint-comments/disable-enable-pair": ["error", {"allowWholeFile": true}],
```

Comment:
This is intended to allow  disabling a rule for the whole file only.  Alternately I can add the disable flag at the end of the file in testConfigvalidation.js.  This seemed like a prudent lint option, but it's definitely up for debate.